### PR TITLE
refactor(stock adjustment): use buttons instead of input radio

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR refactor the stock adjustment ui by using buttons instead of input radio 

stock_adjustment

### CODE REVIEW

File: client/src/modules/stock/adjustment/adjustment.html [hunk 1]
```diff
@@ -37,24 +37,33 @@
   <div class="container-fluid">
     <form name="StockForm" bh-submit="StockCtrl.submit(StockForm)" novalidate>
       <div class="row">
-        <div class="col-xs-6">
+        <div class="col-xs-12 col-sm-6 col-md-4">
```
Comment (jniles) [inline client/src/modules/stock/adjustment/adjustment.html:40]: This is a very nice usability adjustment.  Well done!

File: test/end-to-end/stock/stock.adjustment.page.js [hunk 2]
```diff
@@ -19,8 +19,8 @@ function StockAdjustmentPage() {
    * @method setAdjustment
    * @param {number} radionIndex
    */
-  page.setAdjustment = function setAdjustment(radionIndex) {
-    FU.radio('StockCtrl.adjustmentOption', radionIndex);
+  page.setAdjustment = function setAdjustment(value) {
+    $(`[id=btn-${value}]`).click();
```
Comment (jniles) [inline test/end-to-end/stock/stock.adjustment.page.js]: Alternative, 
```js
element(by.id(`btn-${value}`)).click();
```
which might be slightly faster.

**Review by jniles — COMMENTED** [review]
Code looks fine to me.  Looks like some tests need to be debugged, but otherwise, LGTM.

File: test/end-to-end/stock/stock.adjustment.spec.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ function StockAdjustmentTests() {
 
   it('Should make a positive adjustment ', () => {
     // select the positive adjustment
-    page.setAdjustment(0);
+    page.setAdjustment('increase');
```
Comment (jniles) [inline test/end-to-end/stock/stock.adjustment.spec.js:22]: Nice :+1:

**Review by jniles — APPROVED** [review]
LGTM.

bors r+

**Comment by mbayopanda** [comment]
@jniles could you `bors` this PR again please

**Pull request merged**